config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
```

Comment:
We already have config.js which allows us to use env variables, using config.json would require everyone changing configuration after every commit. Please add this file to your gitignore. Thanks